apps/api/src/domain/models/__tests__/command-helpers/types/FSAFactoryFunction.ts

```diff
@@ -0,0 +1,4 @@
+import { CommandFSA } from '../../../../../app/controllers/command/command-fsa/command-fsa.entity';
+import { AggregateId } from '../../../../types/AggregateId';
+
+export type FSAFactoryFunction = (id: AggregateId) => CommandFSA;
```

Comment:
We know that only for `CREATE_X` commands does the test case need to inject the generated ID. In other commands we've seen so far, there are no arguments for the `FSAFactoryFunction`. Maybe there are cases where there are other arguments as well? We might want to loosen this type.